microprofile/messaging/core/src/main/java/io/helidon/microprofile/messaging/UniversalChannel.java

```diff
@@ -67,6 +69,10 @@ void setOutgoing(OutgoingMethod outgoingMethod) {
         this.outgoingMethod = outgoingMethod;
     }
 
+    AtomicBoolean isUp() {
+        return up;
```

Comment:
Is there a possibility `isUp` is called before `connect`? If so, this would return null and most likely cause a NPE

Reply:
Also `up` is not volatile. Even if the field is AtomicBoolean, access to it must be correctly synchronized if the field may be created in a different thread.
Maybe the AtomicBoolean should be final and true and sent to `Probe.connect` as a parameter

Reply:
oj, thx!